config/crds/edge.kcp.io_edgeplacements.yaml

```diff
@@ -236,6 +241,47 @@ spec:
                   - resources
                   type: object
                 type: array
+              upsync:
+                description: '`upsync` identifies objects to upsync. An object matches
+                  `upsync` if and only if it matches at least one member of `upsync`.'
+                items:
+                  description: 'UpsyncSet specifies a set of objects, which may be
+                    namespaced or cluster-scoped, from one particular API group. An
```

Comment:
we should be consistent wrt replacing 'cluster-scoped' with 'non-namespaced'

Reply:
I think the term cluster-scoped is the one more commonly used in the Kubernetes world.  I sometimes write "non-namespaced" when I particularly want to emphasize the lack of a namespace, which particularly arises in TMC-adjacent work because TMC gives much more emphasis to namespaced objects.

BTW, this file is derived from edge-placement.go, in the future I think it would be better to comment on the source file.

Reply:
ok - its just not consistent with the way we talk about the objects day-to-day.